frontend/components/RestaurantForm.vue

```diff
@@ -126,18 +129,16 @@
           :key="tag"
           :title="tag"
           variant="dark"
-          class="mr-1 tag selected"
+          class="mr-1 tag wr-button main"
           @remove="removeTag(tag)"
         >
           <b>{{ tag }}</b>
         </b-form-tag>
       </div>
     </b-form-group>
-    <b-button v-if="addressEntered" type="submit" class="submit">
-      <b>{{ submitText }}</b>
-    </b-button>
-    <b-button v-else type="submit" class="submit">
-      Weiter
+    <b-button type="submit" class="submit wr-button main">
+      <span v-if="!addressEntered">Weiter</span>
```

Comment:
`v-if="addressEntered"` ?

Reply:
I'll do v-if="addressEmpty" to keep the order of the code the same as the order in time